src/main/java/core/basesyntax/figures/figurehandlers/FigureSupplier.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.figures.figurehandlers;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 5;
+    private static final int RANDOM_VALUES_BORDER = 30;
+    private final Figure defaultFigure = new Circle("White", 10);
+
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomIndex = random.nextInt(FIGURES_COUNT) + 1;
```

Comment:
```suggestion
        int randomIndex = random.nextInt(FIGURES_COUNT);
```
just numerate cases from 0 to 4